web/modules/custom/collection/src/Form/JoinCollectionForm.php

```diff
@@ -50,26 +50,48 @@ public function buildForm(array $form, FormStateInterface $form_state, AccountPr
     // confirmation form, disguised as a form submit button. The confirmation
     // form should open in a modal dialog for JavaScript-enabled browsers.
     if (Og::isMember($collection, $user)) {
-      $form['leave'] = [
-        '#type' => 'link',
-        '#title' => $this->t('Leave this collection'),
-        '#url' => Url::fromRoute('collection.leave_confirm_form', [
-          'rdf_entity' => $collection->id(),
-        ]),
-        '#attributes' => [
-          'class' => [
-            'use-ajax',
-            'button',
-            'button--small',
-            'button--blue-light',
-            'mdl-button',
-            'mdl-js-button',
-            'mdl-button--raised',
-            'mdl-js-ripple-effect',
-            'mdl-button--accent',
+      $form['leave_wrapper'] = [
+        '#type' => 'dropbutton',
```

Comment:
Is there a reason not to use the MDL dropdown already available in the site like: https://getmdl.io/components/index.html#menus-section ?
If there's no specific instruction to avoid it, I'd recommend it as it will be consistent with the rest of the dropdowns like this, like in the login, for example.